src/components/ToDoWrapper.vue

```diff
@@ -16,13 +17,16 @@
   import TodoForm from './TodoForm.vue';
   import TodoList from './TodoList.vue';
   import PageHeader from './PageHeader.vue';
+  import PlaceHolder from './PlaceHolder.vue';
 
   const storage = localStorage.getItem('listOfItems');
 
   const listItems = ref<Array<ToDoItem>>(storage ? JSON.parse(storage) : []);
 
   const showForm = ref(false);
 
+  const showPlaceHolder = ref(listItems.value.length === 0);
```

Comment:
You will have to make this a computed at some point, since the ref will not react to the changes of the listItems length, this means that if you have todos added and delete them this fill now change to the state you would expect